calculator.rb

```diff
@@ -0,0 +1,75 @@
+class CalculatorNumbers
```

Comment:
Classes shouldn't be pluralised - the idea is you're making a blueprint and every time you're creating a single instance of that class. e.g. if you wanted to make a class for a person, you call it Person, not People, as each time you are making one new Person with that class.